parts/kubernetesagentcustomdata.yml

```diff
@@ -9,6 +9,7 @@ runcmd:
 - echo "deb {{{dockerEngineDownloadRepo}}} ubuntu-xenial main" | sudo tee /etc/apt/sources.list.d/docker.list
 - "echo \"Package: docker-engine\nPin: version {{{dockerEngineVersion}}}\nPin-Priority: 550\n\" > /etc/apt/preferences.d/docker.pref"
 - apt-get update
+- apt-get install -y ebtables
```

Comment:
I'd suggest doing the conditional logic with bash... in fact I think you basically have to.